pilot/pkg/model/telemetry.go

```diff
@@ -82,8 +82,8 @@ type Telemetries struct {
 	mu                     sync.Mutex
 }
 
-// telemetryKey defines a key into the computedMetricsFilters cache.
-type telemetryKey struct {
+// TelemetryKey defines a key into the computedMetricsFilters cache.
+type TelemetryKey struct {
```

Comment:
why change this?

Reply:
I was getting errors about it being unexported when using cmp.Equal. But I've since addressed that by adding a customer comparer. I can switch it back now. Good catch